stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/attachment/AttachmentDialogStyle.kt

```diff
@@ -0,0 +1,35 @@
+package io.getstream.chat.android.ui.message.input.attachment
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.graphics.drawable.Drawable
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.common.extensions.internal.getDrawableCompat
+
+public data class AttachmentDialogStyle(
+    val pictureAttachmentIcon: Drawable,
+    @Deprecated(message = "Use pictureAttachmentIcon instead", level = DeprecationLevel.WARNING)
+    val pictureAttachmentIconTint: ColorStateList?,
+    val fileAttachmentIcon: Drawable,
+    @Deprecated(message = "Use fileAttachmentIcon instead", level = DeprecationLevel.WARNING)
+    val fileAttachmentIconTint: ColorStateList?,
+    val cameraAttachmentIcon: Drawable,
+    @Deprecated(message = "Use cameraAttachmentIcon instead", level = DeprecationLevel.WARNING)
+    val cameraAttachmentIconTint: ColorStateList?,
+) {
+    public companion object {
+        /**
+         * Creates an [AttachmentDialogStyle] instance with the default Stream Chat icons.
+         */
+        public fun createDefault(context: Context): AttachmentDialogStyle {
```

Comment:
Would be difficult to create otherwise for clients who need to create instances of `AttachmentDialogFragment`.